org.mixedrealitytoolkit.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -157,10 +226,25 @@ public override bool isHoverActive
                     bool hoverActive = base.isHoverActive;
                     if (hoverActive)
                     {
-                        if (xrController is ArticulatedHandController handController)
+                        bool isPalmFacingAway = false;
+
+#pragma warning disable CS0618 // Type or member is obsolete
+#pragma warning disable CS0612 // Type or member is obsolete
+                        if (forceDeprecatedInput)
+                        {
+                            if (xrController is ArticulatedHandController handController)
```

Comment:
Not a fan of nest if-statements.  Can you clean up with &&

Reply:
Sure, no problemo, done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/750/commits/82d77cd3e504e99bff680bf709b24a1996b515eb , resolving.